x-pack/legacy/plugins/siem/public/components/link_to/redirect_to_hosts.tsx

```diff
@@ -11,12 +11,16 @@ import { RedirectWrapper } from './redirect_wrapper';
 
 export type HostComponentProps = RouteComponentProps<{
   hostName: string;
+  search: string;
 }>;
 
 export const RedirectToHostsPage = ({
   match: {
     params: { hostName },
   },
-}: HostComponentProps) => <RedirectWrapper to={hostName ? `/hosts/${hostName}` : '/hosts'} />;
+  location: { search },
```

Comment:
needed to carry over the search param through these redirects to preserve url state